CMakeLists.txt

```diff
@@ -419,8 +419,10 @@ IF(NOT WIN32 AND NOT APPLE)
 ENDIF(NOT WIN32 AND NOT APPLE)
 
 if(WIN32)
-    FILE(GLOB dll_files "${OpenMW_BINARY_DIR}/Release/*.dll")
-    INSTALL(FILES ${dll_files} DESTINATION ".")
+	FILE(GLOB dll_files_debug "${OpenMW_BINARY_DIR}/Debug/*.dll")
```

Comment:
You've got tabs here and in other places, please configure your editor to spaces only.